content/blog/2022/10/2022-10-20-jenkins-election-announcement.adoc

```diff
@@ -0,0 +1,97 @@
+---
+:layout: post
+:title: "Jenkins Board and Officer Elections 2022"
+:tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- kmartens27
+description: >
+    Nomination and voter registration period 2022 now open for Jenkins Board elections.
+links:
+  discourse: true
+opengraph:
+  image: /images/governance/elections/2022/2022-opengraph.png
+---
+
+We are excited to announce the 2022 Jenkins Governance Board elections!
+The nomination and voter registration period is now open.
+Nominations can be submitted for two link:/project/board[governance board] positions, and all five link:/project/team-leads/[officer] positions, namely: Security, Events, Release, Infrastructure, and Documentation.
+
+Thank you to link:https://github.com/halkeye[Gavin Mogan], link:https://github.com/ewelinawilkosz[Ewelina Wilkosz], link:https://github.com/dduportal[Damien Duportal], link:https://github.com/wadeck[Wadeck Follonier], link:https://github.com/alyssat[Alyssa Tong], link:https://github.com/timja[Tim Jacomb], and link:https://github.com/MarkEWaite[Mark Waite] for all of your work over the last term. 
```

Comment:
Could the role of each one be added?

Reply:
Yes, for sure! I just finished adding them in.